pipeline/steps/az-shared-config.yml

```diff
@@ -76,6 +76,8 @@ steps:
         echo "Logged in as oid $oid"
         
         echo "Creating Shared KV $kvName"
+        tags="businessArea=${{ parameters.businessArea }} application=${{ parameters.application }} builtFrom=${{ parameters.builtFrom }} environment=${{ parameters.env }}"
+        echo "businessArea=${{ parameters.businessArea }} application=${{ parameters.application }} builtFrom=${{ parameters.builtFrom }} environment=${{ parameters.env }}"
```

Comment:
This will fail if the tags are not existing. Speak to @adamtodd0  as he wrote an update script that will always work.